src/main/java/com/mathworks/ci/RunMatlabTestsStep.java

```diff
@@ -155,8 +165,10 @@ private String getInputArgs() {
         inputArgs.add("'Test'");
 
         args.forEach((key, val) -> {
-            if(key.equals("SourceFolder") || key.equals("SelectByFolder") && val != null){
+            if(key.equals("SourceFolder") || key.equals("SelectByFolder") && val != null) {
                 inputArgs.add("'" + key + "'" + "," + val);
+            }else if (key.equals("RunInParallel") && val != null){
+                inputArgs.add("'" + key + "'" + "," + Boolean.parseBoolean(val));
```

Comment:
when this will be null ? since its boolean

Reply:
Aah i see its string . But does it make sense to pass string value here ? i can still pass 'ABC'  which will return false . I dont have strong flag on this though